travis/integration_test-script.sh

```diff
@@ -34,109 +29,132 @@ echo "export DD_ADMIN_USER=admin" >> ~/.profile && \
 
 export DD_BASE_URL='http://localhost:8080/'
 
-echo "Running Product type integration tests"
+function fail() {
+    echo "Error: $1 test failed"
+    docker-compose logs --tail="all" uwsgi
+    exit 1    
+}
+
+function succes() {
```

Comment:
It's be nice to correct the typo here :) `success`

Reply:
ty done